1907-10-23.xml

```diff
@@ -0,0 +1,8070 @@
+                            <cell><name>Roon</name></cell>
+                            <cell><measure quantity="8000" unit="ton">8000</measure> Tons</cell>
+                            <cell>about <date when="1907-11-28">28 Nov</date></cell>
+
+                        </row>
+                        <row>
+                            <cell><name>Scharnhorst</name></cell>
+                            <cell><measure quantity="8300" unit="ton">8300</measure> Tons</cell>
+                            <cell>about <date when="1907-11-05">5 Nov</date></cell>
+                            <cell><name>Bulow</name></cell>
+                            <cell><measure quantity="9000" unit="ton">9000</measure></cell>
+                            <cell><date when="1907-12-02">2 Dec</date></cell>
+                        </row>
+                        <row role="label">
+                            <cell cols="3">OUTWARD: for CHINA and JAPAN via SUEZ, ADEN, COLOMBO,
+                                PENANG, SINGAPORE.</cell>
+                            <cell cols="3">For AUSTRALIA via SUEZ, ADEN, COLOMBO.</cell>
+                        </row>
+                        <row>
+                            <cell><name>Pr. Regt. Luitpold</name></cell>
+                            <cell><measure unit="ton" quantity="6000">6000</measure> Tons</cell>
+                            <cell>about <date when="1907-10-28">28 Oct</date></cell>
+                            <cell><name>Brmen</name></cell>
+                            <cell><measure unit="ton" quantity="11600">11600</measure> Tons</cell>
+                            <cell>about <date when="1907-10-18">18 Oct</date></cell>
+                        </row>
+                        <row>
+                            <cell><name>Yark</name></cell>
+                            <cell><measure unit="ton" quantity="9000">9000</measure> Tons</cell>
+                            <cell>about <date when="1907-11-11">11 Nov</date></cell>
+                            <cell><name>Seydlits</name></cell>
+                            <cell><measure unit="ton" quantity="8900">8900</measure> Tons</cell>
+                            <cell>about <date when="1907-11-15">15 Nov</date></cell>
+                        </row>
+                    </table>
+                    <p>FOR FURTHER PARTICULARS APPLY TO THE AGENTS OF THE</p>
+                    <p>NORDDEUTSCHER LLOYD at Cairo, Alexandria, Port-Said and Suez.</p>
+                    <p>OTTO STERZING, Agent In Cairo, Opera Square.</p>
+                    <p>C. H. SCHOELLER, Agent In Alexandria, Cleopatra Lane.</p>
+                    <p>Messrs. THOS. COOK &amp; SON (Egypt) LTD., and CARL STANGENS REISEBUREAN are
+                        anthorised to sell tickets in CAIRO and ALEXANDRIA, <measure type="indexNo"
+                            >31-8-905</measure></p>
+                </div>
+                <div type="advert" xm
```

Comment:
10/23 Line 1769: Don't use div type="feature". Instead, type should be section for Local and General. Also, you need to update some measures of your templates.